### PR TITLE
creating a file

### LINKED ISSUE #25: Create casperplomp hometown file
## Rijen - The Netherlands
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a pull request 
- [ ] Have a conversation
- [ ] get a :+1:
- [ ] :ship: it (aka merge)

### PR DESCRIPTION
**Pull request by CasperPlomp**
Push wasn't working so I started all over
It resolves #25

### CODE REVIEW

File: CP.md [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (gerryderry) [inline CP.md]: sounds divine

Comment (CasperPlomp) [inline CP.md]: hahaha

Comment (CasperPlomp) [inline CP.md]: Should look better now

**Comment by CasperPlomp** [comment]
@gerryderry How about this ?

**Comment by CasperPlomp** [comment]
This is :+1:

**Pull request merged**